src/components/single-line-diagram.js

```diff
@@ -77,9 +80,11 @@ const SingleLineDiagram = (props) => {
                     setSvg({svg: null, metadata: null, error, svgUrl: props.svgUrl});
                 });
         } else {
-            setSvg(nosvg);
+            setSvg(noSvg);
         }
-    }, [props.svgUrl]);
```

Comment:
We shouldn't need to store the full svg just to test if it's the same as the previous one